app/api/api_v1/routers/admin.py

```diff
@@ -522,6 +584,28 @@ def ingest_law_policy(
     )
 
 
+def validate_law_policy_csv(
+    law_policy_csv: UploadFile,
+    db: Session,
+    context: IngestContext,
+    all_results: list[Result],
+) -> tuple[str, str]:
```

Comment:
New validate function returns the file contents (for storing on s3) and the summary message